hack/allure-report.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+podman version 2>/dev/null >/dev/null || NO_DOCKER=$?
```

Comment:
can we make `podman` as default choice but also allow using `docker` through configuring some env var?

Reply:
done